lib/modules/dosomething/dosomething_api/dosomething_api.services.inc

```diff
@@ -222,6 +222,9 @@ function dosomething_api_default_services_endpoint() {
         ),
       ),
       'targeted_actions' => array(
+        'update' => array(
```

Comment:
Not super keen that we're using targeted actions instead of just REST routes, but also aware that the `/users` endpoint is using old approach and not the newer RESTful one.